src/PowerShellEditorServices/Services/PowerShell/Console/ConsoleProxy.cs

```diff
@@ -1,5 +1,5 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license.
+// Copyright (c) Microsoft Corporation.
```

Comment:
Yay, I love that these can be auto-fixed now! Because we did it semi-manually last time in https://github.com/PowerShell/PowerShellEditorServices/pull/1580 and clearly missed some.